spark-cluster/src/main/scala/tech/ytsaurus/spark/launcher/SparkLauncher.scala

```diff
@@ -172,8 +172,7 @@ trait SparkLauncher {
   }
 
   private def readLivyLogs(): String = {
-    val logsPath = f"$livyHome/logs/livy--server.out"
-    f"cat $logsPath".!!
+    Seq("bash", "-c", """cat "$0"/logs/livy-*-server.out""", livyHome).!!
```

Comment:
It was `livy-root-server.out`:

https://github.com/apache/incubator-livy/blob/6097af1cdd536ebbe1d7eacb1513a440a5fa2784/bin/livy-server#L114